dev-python/QtPy/QtPy-1.9.0-r4.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..8} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Abstraction layer on top of PyQt5 and PySide2 and additional custom QWidgets"
+HOMEPAGE="https://github.com/spyder-ide/qtpy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="
+	pyqt5 pyside2 3d bluetooth dbus charts concurrent datavis declarative
```

Comment:
Maybe separate `pyqt5 pyside2` from the rest of the flags, and choose one of them as the default (I guess pyqt5 since that's what upstream defaults to).